carlos-calc/src/App.js

```diff
@@ -0,0 +1,80 @@
+import { Component } from 'react';
+import "./App.css";
+import Keypad from "./Keypad";
+import CalcOutput from "./CalcOutput";
+
+class App extends Component {
+  constructor() {
+    super(); 
+    this.state = {
+      result: "0",
+      operators: "",
+      num1: "",
+      previousNum: ""
+    }
+  } 
+  handleClick = (e) => {
+    const { result } = this.state;
+    this.setState({ result: result + e.target.value });
```

Comment:
Best practice is when using the current state to build the next state to use a callback function inside setState